ostd/src/io_mem.rs

```diff
@@ -2,27 +2,43 @@
 
 //! I/O memory.
 
-use core::ops::Range;
+use core::ops::{Deref, Range};
+
+use align_ext::AlignExt;
+use cfg_if::cfg_if;
 
 use crate::{
     mm::{
-        kspace::LINEAR_MAPPING_BASE_VADDR, paddr_to_vaddr, FallibleVmRead, FallibleVmWrite,
-        HasPaddr, Infallible, Paddr, PodOnce, Vaddr, VmIo, VmIoOnce, VmReader, VmWriter,
+        kspace::kva::{Kva, UnTracked},
+        page_prop::{CachePolicy, PageFlags, PageProperty, PrivilegedPageFlags},
+        HasPaddr, Paddr, PodOnce, VmIo, VmIoOnce, VmReader, VmWriter, PAGE_SIZE,
     },
     prelude::*,
-    Error,
 };
 
+cfg_if! {
+    if #[cfg(feature = "cvm_guest")] {
+        use tdx_guest::tdx_is_enabled;
+    }
+}
+
```

Comment:
The `tdx_is_enabled` function is used only once. So remove the `use` statement here and add it to the place using it.